src/tests/ftest/erasurecode/restart.py

```diff
@@ -28,46 +73,131 @@ def execution(self, agg_check=None):
 
         Args:
             agg_check: When to check Aggregation status.Either before restarting all the servers or
-                       after.Default is None so not to check aggregation, but wait for 20 seconds
-                       and restart the servers.
+                       after. Default to None.
         """
-        # Write all EC object data to NVMe
-        self.ior_write_dataset(operation="Auto_Write", percent=self.percent)
-        self.log.info(self.pool.pool_percentage_used())
-        # Write all EC object data to SCM
-        self.ior_write_dataset(storage='SCM', operation="Auto_Write", percent=self.percent)
-        self.log.info(self.pool.pool_percentage_used())
-        self.pool.pool_percentage_used()
-
-        if not agg_check:
-            # Set time mode aggregation
-            self.pool.set_property("reclaim", "time")
-            # Aggregation will start in 20 seconds after it sets to time mode. So wait for 20
-            # seconds and restart all the servers.
-            time.sleep(20)
-
-        if agg_check == "Before":
-            # Verify if Aggregation is getting started
-            if not any(self.check_aggregation_status(attempt=50).values()):
-                self.fail("Aggregation failed to start Before server restart..")
-
-        # Shutdown the servers and restart
+        # 1.
+        ior_w_flags = self.params.get("flags", "/run/ior/iorflags/*")
+        ior_read_flags = self.params.get("read_flags", "/run/ior/iorflags/*")
+        block_transfer_sizes = self.params.get("block_transfer_sizes", "/run/ior/*")[0]
+        processes = self.params.get("np", "/run/ior/client_processes/*", default=2)
+        ppn = self.params.get("ppn", "/run/ior/client_processes/*")
+        obj_class = self.params.get("dfs_oclass_list", "/run/ior/objectclass/*")
+        aggr_threshold = float(self.params.get("threshold", "/run/aggregation/*")[:-1]) / 100
+        block_size = block_transfer_sizes[0]
+        transfer_size = block_transfer_sizes[1]
+        self.log_step("Create pool and containers")
+        self.pool = self.get_pool()
+        containers = {}
```

Comment:
Let's simplify this as the oclass value can be obtained from the `TestContainer.oclass.value`:
```suggestion
        containers = []
```